ddtrace/internal/telemetry/data/application.py

```diff
@@ -0,0 +1,52 @@
+import sys
+from typing import TypedDict
+
+from ....settings import _config as config  # noqa: E402
+from ....version import get_version
+
+
+Application = TypedDict(
+    "Application",
+    {
+        "service_name": str,
+        "service_version": str,
+        "env": str,
+        "language_name": str,
+        "language_version": str,
+        "tracer_version": str,
+        "runtime_name": str,
+        "runtime_version": str,
+    },
+)
+"""
+Stores information to uniquely identify a service
+"""
```

Comment:
module docstrings should go at the top of the file

Reply:
This is a docstring for the the application TypedDict and not the module. I did this for all the typeddicts I defined in this change. Should I change those doc strings to a module doc string?

Reply:
You should just add as a comment on the line before.

we do the multiline string as a comment for modules and functions/class/methods because it'll get added to the `__doc__` property of the function/class/method.


```python
def func():
    """this is a function docstring"""
    pass

print(func.__doc__)
```

The way you have it doesn't do that, so you can just do a normal comment on the line before it.